drivers/pinctrl/intel/pinctrl-intel.c

```diff
@@ -74,6 +75,7 @@
 #define PADCFG2_DEBOUNCE_MASK		GENMASK(4, 1)
 
 #define DEBOUNCE_PERIOD			31250 /* ns */
+#define PINCTRL_FEATURE_INT_CHECK_TIMER	BIT(8)
```

Comment:
please move this to the other PINCTRL_FEATURE_ definitions so that there's less risk of us not noticing bit 8 being used with another meaning in future rebases

Reply:
I'll move it to pinctrl-intel.h